README.rst

```diff
@@ -82,6 +64,21 @@ This command will display the external IP address of the EC2 instance.
 
 Run ``python start-aws.py --help`` to see the available options to this command.
 
+Create a staging Docker image
+=============================
+
+On the staging server, run the following commands::
+
+   [aws]$ sudo yum install -y docker-io fabric git
+   [aws]$ sudo setenforce 0
+   [aws]$ sudo systemctl start docker
+   [aws]$ git clone https://github.com/ClusterHQ/build.clusterhq.com.git
+   [aws]$ cd build.clusterhq.com
+   [aws]$ # Change <BRANCH> to the branch of build.clusterhq.com you want
+   [aws]$ git checkout <BRANCH>
+   [aws]$ sudo docker build -t clusterhq/build.clusterhq.com:staging .
+   [aws]$ sudo docker run --name buildmaster-data -v /srv/buildmaster/data busybox /bin/true
```

Comment:
I never did this step. How important is it? If it's important, add an explanatory note. I guess it wipes out the data from the existing staged build slave (if there is one on this staging server)